tests/foreman/ui/test_contentview.py

```diff
@@ -2273,11 +2277,113 @@ def test_positive_subscribe_system(self):
 
         @assert: Systems can be subscribed to content view(s)
 
-        @caseautomation: notautomated
-
-
         @CaseLevel: Integration
         """
+        cv_name = gen_string('alpha')
+        filter_name = gen_string('alpha')
+        subscription_name = DEFAULT_SUBSCRIPTION_NAME
+        puppet_repo_url = FAKE_0_PUPPET_REPO
+        puppet_module_name = 'httpd'
+        repos = [
```

Comment:
basically NACK overall idea to put completely different functionalities into one test case. Divide it into three test cases and remove that complex logic